db-server/index.js

```diff
@@ -177,6 +176,14 @@ function createServer(db) {
     }
   }
 
+  api.get('/account/:id/devices', withIdAndBody(db.accountDevices))
+
+  api.get(
+    '/account/:uid/device/:deviceId',
+    op(function (req) {
+      return db.device(req.params.uid, req.params.deviceId)
```

Comment:
Not sure if it makes sense or is the correct convention, ignore if not, but can this and the called method be tweaked to use the `withParams` helper here?

Reply:
I tried this out, but it made the tests uglier because you have to call e.g. `db.createDevice({uid: account.uid, deviceId: deviceinfo.id }, deviceInfo)`.  I'm adding a modified helper that I like better, but YMMV, I'd be interested to hear what you think.